packages/gatsby/src/redux/actions/public.ts

```diff
@@ -1076,7 +1101,10 @@ actions.createParentChildLink = (
  *
  * @param {Object} config partial webpack config, to be merged into the current one
  */
-actions.setWebpackConfig = (config: Object, plugin?: ?Plugin = null) => {
+actions.setWebpackConfig = (
+  config: Record<string, any>,
```

Comment:
I think this should be something like `webpack.Config` imported from webpack.

Reply:
Done in 3ccf334